typed-racket-lib/typed-racket/utils/tc-utils.rkt

```diff
@@ -6,9 +6,10 @@ don't depend on any other portion of the system
 |#
 
 (require syntax/source-syntax "disappeared-use.rkt"
-         racket/promise racket/string
-         syntax/parse/pre (for-syntax racket/base syntax/parse/pre)
-         (only-in unstable/sequence in-slice))
+         racket/promise racket/string racket/lazy-require
+         syntax/parse/pre (for-syntax racket/base syntax/parse/pre))
+
+(lazy-require [unstable/sequence (in-slice)])
```

Comment:
The point of this commit is for it not to be `require`d at runtime, so I don't think there's a problem here. I expect that the typechecker itself depends on `unstable/syntax` in a variety of other ways too.

The better solution would be to split `tc-utils` into a module needed only in the type checker and one that gets used by other modules.